src/containers/System/components/Manufacturer/index.module.scss

```diff
@@ -0,0 +1,20 @@
+.Manufacturer {
+  :global(.ant-card) {
+    margin-top: 10px;
+  }
+  .fileButton {
+    margin-top: 20px;
+  }
+  .FlexDiv {
+    display: flex;
+    justify-content: center;
+    margin-left: 30px;
+  }
+
+  .Field {
+    background: #141414;
+    border: 1px solid #434343;
+    border-radius: 5px;
+    margin-bottom: 5px;
+  }
```

Comment:
I created a global .field class use it by doing 
`import globalStyles from 'styles/index.scss';` and use by `globalStyles.field`